build.gradle

```diff
@@ -132,13 +132,14 @@ configurations {
 }
 
 dependencies {
+    compileOnly group: 'org.opensearch', name:'opensearch-security-spi', version:"${opensearch_build}"
     implementation "org.opensearch:opensearch:${opensearch_version}"
     compileOnly "org.opensearch.plugin:opensearch-scripting-painless-spi:${opensearch_version}"
     compileOnly "org.opensearch:opensearch-job-scheduler-spi:${job_scheduler_version}"
     implementation "org.opensearch:common-utils:${common_utils_version}"
     implementation "org.opensearch.client:opensearch-rest-client:${opensearch_version}"
-    implementation group: 'com.google.guava', name: 'guava', version:'32.1.3-jre'
-    implementation group: 'com.google.guava', name: 'failureaccess', version:'1.0.2'
+    implementation group: 'com.google.guava', name: 'guava', version:'33.4.5-jre'
```

Comment:
Can we use the version from Core: https://github.com/opensearch-project/OpenSearch/blob/main/gradle/libs.versions.toml#L23 ? So that all plugin can have the same version and not to introduce version Conflict?